scripts/publish/aws-s3.sh

```diff
@@ -30,21 +30,24 @@ function usage() {
   echo "    -f <file>"
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
-  echo "    -p <product name>"
+  echo "    -n <product name>"
+  echo "    -p S3 key prefix"
```

Comment:
So we can use p for **p**refix. I guess we can also keep `-p` for product name, and use `-x` for the prefix. What do you think?